nodes/src/nodes/dataset_cobalt/dataset_loader.py

```
@@ -0,0 +1,263 @@
+            items: List of dicts, each representing a dataset item.
+
+        Returns:
+            List of dataset item dicts.
+
+        Raises:
+            ValueError: If items is empty or not a list.
+        """
+        from cobalt import Dataset
+
+        if not items or not isinstance(items, list):
+            raise ValueError('Inline items must be a non-empty list of dicts')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)